docs/translations/translations.json

```diff
@@ -97,10 +97,9 @@
   "feedbackMessageUp": "What did you like about this page? (optional)",
   "feedbackSectionSpecific": "How can we improve the <strong>{{sectionName}}</strong> section? (optional)",
   "feedbackMessageToGitHub": {
-    "usecases": "Is something broken? Do you need a reply to a problem you've encountered?",
+    "usecases": "If something is broken or if you need a reply to a problem you've encountered, please",
     "callToAction": {
```

Comment:
```suggestion
    "reasonWhy": "Otherwise team will not be able to answer you back or ask for additional information.",
    "callToAction": {
```

Reply:
I'm wondering if we should insist on the fact their is no way the be contacted back

Reply:
Good call!